src/services/services.ts

```diff
@@ -3113,7 +3114,9 @@ namespace ts {
             }
             else if (isRightOfOpenTag) {
                 let tagSymbols = typeChecker.getJsxIntrinsicTagNames();
-                if (tryGetGlobalSymbols()) {
+                // If the currect cursor is inside JSX opening tag, the only meaningful completions are those of JSX.IntrinsicElements or users defined React.Component
+                // If the services can't find those symbols, then show nothing instead of including all the global symbols in the completion list.
+                if (tryGetGlobalSymbols(/*includeAllGlobalSymbols*/false)) {

```

Comment:
I agree on naming. not sure what is the best name.  What I try to convey is that the boolen control whether you will add all global symbols from all the files into the completions list or not. This doesn't make sense for JSX opening tag